lib/glific/contacts/import.ex

```diff
@@ -12,22 +12,64 @@ defmodule Glific.Contacts.Import do
       phone: data["phone"],
       organization_id: organization_id,
       language_id: Enum.at(Settings.get_language_by_label_or_locale(data["language"]), 0).id,
-      optin_time: elem(Timex.parse(data["opt_in"], "{YYYY}-{0M}-{0D}"), 1)
+      optin_time:
+        if(String.length(data["opt_in"]) != 0,
```

Comment:
data["opt_in"] != ""

Reply:
Yeah I was unsure if this is enough. But I think it is. Gonna update it